docs/en/ingest-management/fleet/fleet-server-scaling.asciidoc

```diff
@@ -0,0 +1,300 @@
++
+[role="screenshot"]
+image::images/fleet-server-configuration.png[{fleet-server} configuration]
+
+[discrete]
+[[fleet-server-configuration]]
+=== Advanced {fleet-server} options
+
+The following advanced settings are available to fine tune your {fleet-server}
+deployment.
+
+//TODO: We need to discuss the best way to format config settings. I'm using
+//mostly tables in other sections of the docs for improved scanning, but that is
+//controversial.
+
+//Nima: I tried removed repetitive words and tried clarify the descriptions here,
+//but there were some inconsistencies. You'll want to check this carefully.
+
+`cache`::
+
+`num_counters`:::
+Size of the hash table. Best practice is to have this set to 10x max
+connections.
+
+`max_cost`:::
+Total size of the cache.
+
+`server.limits`::
+`policy_throttle`:::
+How often a new policy is rolled out to the agents.
+
+`checkin_limit.interval`:::
+How fast the agents can check in to the {fleet-server}.
+
+`checkin_limit.burst`:::
+Burst of check-ins allowed before falling back to the rate defined by
+`interval`.
+
+`checkin_limit.max`:::
+Maximum number of agents.
+
+`artifact_limit.max`:::
+Maximum number of agents that can call the artifact API concurrently. It allows
+the user to avoid overloading the {fleet-server} from artifact API calls.
+
+`artifact_limit.interval`:::
+How often artifacts are rolled out. Default of 100ms allows 10 artifacts to be
+rolled out per second.
+
+`artifact_limit.burst`:::
+Number of transactions allowed for a burst, controlling oversubscription on
+outbound buffer.
+
+`ack_limit.max`:::
+Maximum number of agents that can call the Ack API concurrently. It allows the
+user to avoid overloading the {fleet-server} from Ack API calls.
+
+`ack_limit.interval`:::
+How often an acknowledgment (ACK) is sent. Default value of 10ms enables 100
+ACKs per second to be sent.
+
+`ack_limit.burst`:::
+Burst of ACKs to accommodate (default of 20) before falling back to the rate
+defined in `interval`.
+
+`enroll_limit.max`:::
+Maximum number of agents that can call the Enroll API concurrently. This setting
+allows the user to avoid overloading the {fleet-server} from Enrollment API
+calls.
+
+`enroll_limit.interval`:::
+Interval between processing enrollment request. Enrollment is both CPU and RAM
+intensive, so the number of enrollment requests needs to be limited for overall
+system health. Default value of 100ms allows 10 enrollments per second.
+
+`enroll_limit.burst`:::
+Burst of enrollments to accept before falling back to the rate defined by
+`interval`.
+
+[discrete]
+[[scaling-recommendations]]
+=== Scaling recommendations
+
+The following tables provide resource requirements and scaling guidelines based
+on the number of agents required by your deployment:
+
+* <<resource-requirements-by-number-agents>>
+* <<recommend-settings-scaling-agents-a>>
+* <<recommend-settings-scaling-agents-b>>
+
+// Nima: Are these guidelines all cloud-specific? What about on prem/self-managed?
```

Comment:
yes all cloud specific. We are working on getting the same for on-prem

Reply:
I'm going to add Elastic Cloud to the title for now to make this clear.